### ISSUE TITLE
Add account creation via ELIXIR to wiki

### ISSUE THREAD

**Issue by keesvanbochove** [EHDEN/Portal#80]
Currently, the account creation instructions (https://github.com/EHDEN/Portal/wiki/Account-Creation) don't mention yet the possibility of creating an account via ELIXIR, we should update it to reflect this. I'm happy to help if someone could give me edit rights on the wiki?

**Comment by joaorafaelalmeida** [comment]
I think @PRijnbeek is the person with permission for managing this repository, i.e., give you rights to edit the wiki.

**Comment by PRijnbeek** [comment]
agree, but the question is where should this information live. I am not sure about the wiki for this but like to hear opinions. It definitely has to be a knowledge item in the service desk.

**Comment by PRijnbeek** [comment]
@keesvanbochove  was not part of The Hyve group and therefore did not had write rights.
